ci/check-documentation.py

```diff
@@ -114,8 +114,25 @@ def complain(message):
             if md_path in index_child_md_links:
                 referenced_md_files.add(target_path)
 
-        # check for existence of links to README.md files of models and demos
+        # check <omz_dir> link validity
+
+        for link in sorted([link for link in doc_page.omz_references() if link.startswith('<omz_dir>')]):
+            file_path = Path(link.replace('<omz_dir>', str(OMZ_ROOT)))
+
+            try:
+                file_relative_path = file_path.relative_to(OMZ_ROOT)
+            except ValueError:
+                complain(f'{md_path_rel}: invalid OMZ reference {file_path!r}')
+                continue
+
```

Comment:
It would be useful to check whether the reference points outside the OMZ directory, like we do for URLs. (_Unlike_ URLs, the `<omz_dir>` references are always rooted at the OMZ root, so it would be sufficient to check whether the path contains any `..` components.)